src/Components/DynamicFields.res

```diff
@@ -470,6 +470,7 @@ let make = (
                 optionalRequiredFields={Some(requiredFields)}
               />
             </>
+          | CurrencyNetworks => <CryptoCurrencyNetworks />
```

Comment:
Could you please rename the enum as CryptoCurrencyNetworks or the file name as CurrencyNetworks. Let's keep this enum either as generic or specific to improve code readability

Reply:
Updated